From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-huge_memory-refactor-enabled_store-with-change_enabled-checkpatch-fixes
Date: Tue Mar 10 11:37:46 AM PDT 2026


ERROR: code indent should use tabs where possible
#78: FILE: mm/huge_memory.c:360:
+^I^I^I^I^I^I        &transparent_hugepage_flags);$

total: 1 errors, 0 warnings, 78 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/mm-huge_memory-refactor-enabled_store-with-change_enabled.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Breno Leitao <leitao@debian.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/huge_memory.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/huge_memory.c~mm-huge_memory-refactor-enabled_store-with-change_enabled-checkpatch-fixes
+++ a/mm/huge_memory.c
@@ -354,10 +354,10 @@ static bool set_global_enabled_mode(enum
 	for (m = 0; m < ARRAY_SIZE(thp_flags); m++) {
 		if (m == mode)
 			changed |= !__test_and_set_bit(thp_flags[m],
-						       &transparent_hugepage_flags);
+						&transparent_hugepage_flags);
 		else
 			changed |= __test_and_clear_bit(thp_flags[m],
-						        &transparent_hugepage_flags);
+						&transparent_hugepage_flags);
 	}
 
 	return changed;
_
